config/routes.rb

```diff
@@ -40,7 +41,8 @@
   resources :donations, only: [:index, :new, :create]
 
   get 'past-events', to: 'events#past_events', as: 'past_events'
-  root 'home#home'
```

Comment:
I'm not sure if I like this change - I'd rather you just load the photos into the existing home page to keep all the pages named properly. Not a big deal but i think it's a pretty easy fix.

Reply:
fixed